sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIOIT.java

```diff
@@ -137,27 +153,74 @@ public void readUsingHadoopInputFormat() {
         .apply("Generate sequence", GenerateSequence.from(0).to(numberOfRows))
         .apply("Produce db rows", ParDo.of(new DeterministicallyConstructTestRowFn()))
         .apply("Prevent fusion before writing", Reshuffle.viaRandomKey())
+        .apply("Record write time", ParDo.of(new TimeMonitor<>(NAMESPACE, "write_time")))
```

Comment:
We used to call this step "Collect ... " in other IOITs before. Can we call it like this here too?

Reply:
Done